ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/ContentTest.kt

```diff
@@ -291,6 +291,17 @@ class ContentTest : ClientLoader(5 * 60) {
         }
     }
 
+    @Test
+    fun testDuplicatedContentType() = clientTests(listOf("CIO", "Android", "Apache", "Curl", "Java", "Jetty", "Okhttp")) {
```

Comment:
Is there a reason to ignore all these engines?

Reply:
because only browserjs and Darwin engine have this bug

Reply:
But it's still good to verify this behaviour for other engines.